### DISCUSSION TITLE
Is there a way to enable exceptions caught by Sentry to log to the standard Console?

### DISCUSSION THREAD

**Discussion by mikeryder** [getsentry/sentry-dart#2212] [Q&A]
When running my app in certain environments, I would like exceptions to be logged to the console as well as reported to Sentry.  I have it working with one or the other.

I am new to Sentry, but I have scoured the documentation and played with overriding the FlutterError.onError and adding a call to options.beforeSend, but none seem to be doing what I want them to do.  

Thanks for your help!

**Comment by buenaflor** [comment]
hi, why did using `beforeSend` not work?

you should have access to `event.throwable`

**Reply by mikeryder** [reply]
I was hoping for a solution that would show the full exception including stacktrack, etc. on the console.  It feels like there should be an easy way to do that.
